domains/tokyo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A-Tokyo's website",
+  "repo": "https://github.com/a-tokyo/tokyo",
+  "owner": {
+    "username": "a-tokyo",
+    "email": "ahmed.tokyo1@gmail.com"
+  },
+  "record": {
+    "CNAME": "tokyo.is-a-dev.github.io",
```

Comment:
This seems to be an invalid value. I think what you're looking for is `tokyo.github.io`. If you want help with the github pages, you can have a look at [their doc](https://guides.github.com/features/pages/)

Reply:
@phenax Done! Thanks!